reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Jobs.md

```diff
@@ -1,371 +1,415 @@
 remote computer, and the job results are maintained on the remote computer.
-From the perspective of the job, all operations are local. You are just
-running commands remotely to manage a local job on the remote computer.
-
-#### STEP 1: INVOKE-COMMAND START-JOB
-
-Use the Invoke-Command cmdlet to run a Start-Job command on a remote computer.
-
-This command requires a PSSession (a persistent connection). If you use the
-ComputerName parameter of Invoke-Command to establish a temporary connection,
-the Invoke-Command command is considered to be complete when the job object is
-returned. As a result, the temporary connection is closed, and the job is
-canceled.
-
-The following command uses the New-PSSession cmdlet to create a PSSession that
-is connected to the Server01 computer. The command saves the PSSession in the
-\$s variable.
+From the perspective of the job, all operations are local. You are just running
+commands remotely to manage a local job on the remote computer.
+
+1. Use the `Invoke-Command` cmdlet to run a `Start-Job` command on a remote
+   computer.
+
+   This command requires a PSSession (a persistent connection). If you use the
+   ComputerName parameter of `Invoke-Command` to establish a temporary
+   connection, the `Invoke-Command` command is considered to be complete when
+   the job object is returned. As a result, the temporary connection is closed,
+   and the job is canceled.
+
+   The following command uses the `New-PSSession` cmdlet to create a PSSession
+   that is connected to the Server01 computer. The command saves the PSSession
+   in the `$s` variable.
+
+   ```powershell
+   $s = New-PSSession -computername Server01
+   ```
+
+   The next command uses the `Invoke-Command` cmdlet to run a `Start-Job`
+   command in the PSSession. The `Start-Job` command and the `Get-Eventlog`
+   command are enclosed in braces.
+
+   ```powershell
+   Invoke-Command -session $s -scriptblock {
+     Start-Job -scriptblock {Get-Eventlog system}}
+   ```
+
+   The results resemble the following sample output.
+
+   ```Output
+   Id       Name    State      HasMoreData     Location   Command
+   --       ----    -----      -----------     --------   -------
+   2        Job2    Running    True            Localhost  Get-Eventlog system
+   ```
+
+   When you run a `Start-Job` command remotely, `Invoke-Command` returns the
+   same type of job object that `Start-Job` returns. You can save the job
+   object in a variable, or you can use a `Get-Job` command to get the job.
+
+   Note that the value of the Location property shows that the job ran on the
+   local computer, known as "LocalHost", even though the job ran on the
+   Server01 computer. Because the job object is created on the Server01
+   computer and the job runs on the same computer, it is considered to be a
+   local background job.
+
+1. To manage a remote background job, use the Job cmdlets. Because the job
```

Comment:
Should "Job cmdlets" be "`Job-*` cmdlets"?

Reply:
stet